doc/tutorials/3d/point_cloud/point_cloud.markdown

```diff
@@ -0,0 +1,75 @@
+Point cloud visualisation {#tutorial_point_cloud}
+==============================
+
+|    |    |
+| -: | :- |
+| Original author | Klepikov Dmitrii |
```

Comment:
Name usually comes first in English. It's better to write `Dmitrii Klepikov` to exclude confuses in emails and other communication.